### DISCUSSION TITLE
Autodiscover does not detect all repositories

### DISCUSSION THREAD

**Discussion by bobvandevijver** [renovatebot/renovate#8752] [Request Help] [answered]
The autodiscover functionality does not seem to discover all repositories the user has access to, most notably, it seems to skip the repositories that contain the renovate config. All other repositories that the user has access to are detected correctly.

```
 INFO: Autodiscovered repositories
       "length": 6,
       "repositories": [
         "intern/runner-templates",
         "extern/023-xxx",
         "extern/007-xxx",
         "extern/015-xxx",
         "extern/016-xxx",
         "extern/013-xxx"
       ]
```

I'm specifically missing the `intern/renovate-runner` repo, as I would like renovate to update it's own image in the gitlab-ci file.

Am I missing anything?

**Comment by viceice** [comment, marked as answer]
Please provide some more details, did you manually checked the access?

we use a filter, see

https://github.com/renovatebot/renovate/blob/fa9031f5e9f2d887adb289ad54b694020d43c9ca/lib/platform/gitlab/index.ts#L120

**Reply by bobvandevijver** [reply]
Okay, now I feel stupid... The merge requests were disabled for the repository 😅
